tests/apps/actordotnet/CarActor.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace DaprDemoActor
+{
+    using Dapr.Actors.Runtime;
+    using System.Threading.Tasks;
+    using System.Text.Json;
+
+    [Actor(TypeName = "DotNetCarActor")]
+    public class CarActor : Actor, ICarActor
```

Comment:
Should the test include serializing references as well? It's a fairly common pattern for actors to pass references/ids around, and that's one of the parts of the system that really needs to interop well across SDKS.

Reply:
I can test it but it is just to serialize and the reference does not stick on the other end.

Reply:
> the reference does not stick on the other end

I'm not quite sure what you mean by that, care to elaborate?

So that pattern that I'm referring to is that a message sent to/from an actor might include a reference to another actor, as a *callback*. It's a way to say - "hey, do this job, and notify this other actor when you finish". This sort of deferred messaging pattern is common because sending messages to other actors while processing a message yourself can lead to deadlocks.

From an E2E POV here's what's important:

- Each SDK has it's own data type for ActorId and ActorReference 
- They need to be able to share these values as data
- Therefore they all need to produce the same wire format (same JSON)
- We should verify that all of the SDKs can read/write ActorId and ActorReference in a way that's compatible with the others

Reply:
I guess we must get into a call here because I am not sure what you mean.

Reply:
Talked offline - the python and Java SDKs don't yet provide a data type for Actor references, and the actor ID is just a string in its wire format - so there's nothing to test (for now)

Reply:
I see what you mean now. We don't have `ActorReference` in Java or Python SDKs. I've created issues to track that:
https://github.com/dapr/java-sdk/issues/447
https://github.com/dapr/python-sdk/issues/170